cypress/integration/24-authentication/user-logs-in/user_logs_in.js

```diff
@@ -1,21 +1,32 @@
-import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';
-import { USER_CONNECT_ORCID, USER_NO_ARP, USER_NO_NAME_IN_ARP, USER_WITH_AUTHOR } from '../../../support/constants';
+import { Given, When, Then, And, After } from 'cypress-cucumber-preprocessor/steps';
+import {
+  USER_CONNECT_ORCID,
+  USER_NO_ARP,
+  USER_NO_NAME_IN_ARP,
+  USER_WITH_AUTHOR,
+  USER_NAME_IN_ARP,
+} from '../../../support/constants';
 
 Given('that the user logs in with their Feide ID', () => {});
 
-// @217
-// Scenario: User without their Feide ID in ARP logs in
-And('they do not have their Feide ID in any ARP entry', () => {
-  cy.login(USER_NO_ARP);
-});
+// Common steps for @217 and @219
 Then(
   'they see a list containing "Name", "Last registration" and "Institutions" for each ARP entry matching their name',
   () => {
+    if (window.testState.currentScenario.name === 'User without their Feide ID in ARP logs in') {

```

Comment:
dette blir koselig når scenario bytter navn osv 😭 
men bra du fant en løsning da!

Reply:
Skal se nærmere på det, men følte et behov for å avslutte denne jobben
Det er noen flere muligheter der som jeg skal sjekke